File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/how-to--programmatically-add-an-entry-to-outlook-contacts_1.vb
    Private Sub AddContact()
        Dim newContact As Outlook.ContactItem = Me.Application.CreateItem(Outlook. _
            OlItemType.olContactItem)
        Try
            With newContact
                .FirstName = "Jo"
                .LastName = "Berry"
                .Email1Address = "somebody@example.com"
                .CustomerID = "123456"
                .PrimaryTelephoneNumber = "(425)555-0111"
                .MailingAddressStreet = "123 Main St."
                .MailingAddressCity = "Redmond"
                .MailingAddressState = "WA"
                .Save()
                .Display(True)
            End With
        Catch
            MsgBox("The new contact was not saved.")
        End Try
    End Sub

File: OLSource1/v1/VS_officedev/codesnippet/CSharp/how-to--programmatically-add-an-entry-to-outlook-contacts_1.cs
        private void AddContact()
        {
            Outlook.ContactItem newContact = (Outlook.ContactItem)
                this.Application.CreateItem(Outlook.OlItemType.olContactItem);
            try
            {
                newContact.FirstName = "Jo";
                newContact.LastName = "Berry";
                newContact.Email1Address = "somebody@example.com";
                newContact.CustomerID = "123456";
                newContact.PrimaryTelephoneNumber = "(425)555-0111";
                newContact.MailingAddressStreet = "123 Main St.";
                newContact.MailingAddressCity = "Redmond";
                newContact.MailingAddressState = "WA";
                newContact.Save();
                newContact.Display(true);
            }
            catch
            {
                MessageBox.Show("The new contact was not saved.");
            }
        }